mycore-iview2/src/main/java/org/mycore/iview2/services/MCRIView2Tools.java

```diff
@@ -206,9 +206,29 @@ public static boolean isTiled(MCRPath file) {
      * @throws JDOMException
      *             if image properties could not be parsed.
      */
-    public static BufferedImage getZoomLevel(Path iviewFile, int zoomLevel) throws IOException, JDOMException {
-        ImageReader reader = getTileImageReader();
+    public static BufferedImage getZoomLevel(Path iviewFile, int zoomLevel)
+        throws IOException, JDOMException {
```

Comment:
First check if iviewFile is a directory, handle it that way. Else open FileSystem and call the same method again with the root directory of that filesystem...